Second/src/Second.java

```diff
@@ -3,7 +3,11 @@
 
 	public static void main(String[] args) {
 		// TODO Auto-generated method stub
+
+ System.out.println("Hello world -second branch");
+
  System.out.println("Hello world -feature branch Ventatesh changed this file");
+
 	}
 
 }
```

Comment:
Create another method for your changes

Reply:
Created method newMessage() as requested

Reply:
Where are test cases  please add test cases.